planning/behavior_path_planner/autoware_behavior_path_static_obstacle_avoidance_module/README.md

```diff
@@ -83,7 +83,7 @@ partition fillEgoStatus() {
 note right
   This module has following status:
   - RUNNING: target object is still remaining. Or, the ego hasn't returned original lane.
-  - CANCEL: taget obejct has gone. And, the ego hasn't initiated avoidance maneuver.
+  - CANCEL: taget object has gone. And, the ego hasn't initiated avoidance maneuver.
```

Comment:
```suggestion
  - CANCEL: target object has gone. And, the ego hasn't initiated avoidance maneuver.
```

Reply:
@go-sakayori Thanks! I fixed it in https://github.com/autowarefoundation/autoware.universe/pull/7554/commits/dcdb6f5acf2ada1c030b31ffd16e92b56ca13c9c.